test/parallel/test-fs-timestamp-parsing-error.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+require('../common');
+var assert = require('assert');
+var fs = require('fs');
+
+[undefined, null, {}, []].forEach((input) => throws(input));
+
+function throws(input) {
+  assert.throws(() => fs._toUnixTimestamp(input)
+  , 'Error: Cannot parse time: ' + input);
```

Comment:
Alternatively, make this second argument a regex... `new RegExp(`^Error: Cannot parse time: ${input}$`)`